src/components/DependencyDetailsTable/DependencyDetailsTable.js

```diff
@@ -40,15 +40,28 @@ class DependencyDetailsTable extends Component<Props> {
         <tbody>
           <tr>
             <FirstCell>
-              <Label>Last Published</Label>
+              <Label>Location</Label>
```

Comment:
It'd be great to add a little `?` icon here, that links to somewhere that explains what this is.

Additionally, I'm not sure it should link to the official NPM docs, since NPM docs assume a Node environment, where the difference is more stark.

This might be too much trouble, so please don't feel obligated, but I think we should write a little markdown doc that covers:

- They're two different sections of `package.json`
- Historically, and for other types of applications, `dependencies` would be the ones used within the application logic, while `devDependencies` would only be used by developers outside of the application code. Maybe throw in an example for things like running automated tests, deploying, etc
- For front-end React apps, only the dependencies your code uses will be included in the bundle, regardless of whether they're installed in `dependencies` or `devDependencies`
- The distinction might still be helpful purely for organizational purposes, to help you and other developers understand how dependencies are used.
- A link to the NPM docs for additional reading

It could live in `docs/`, alongside the other readmes.

---

We already have a little `?` icon - it's used inside `Module.js` to explain the major modules. It would actually be good to pull that `HelpButton` styled-components component into its own file, and make it take props for `size` and `href`:

```js
type Props = {
  size?: number,
  href: string
}

const HelpButton = ({ size = 18, href }) => (
  <HelpButtonElement 
    size={size} 
    onClick={() => {
      shell.openExternal(href);
    }
  >
    <IconBase icon={questionMarkIcon} size={size - 4} />
  </HelpButtonElement>
)

const HelpButtonElement = styled.button`
  width: ${props => props.size}px;
  height: ${props => props.size}px;
  // copy other styles
`
```

-----

Between writing the doc and doing this refactor, this requested change is a lot of work! If you don't have the time or inclination to tackle this right now, no worries :) just let me know and I'll copy this text to a new issue, it can be tackled later.